webpack.config.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable linebreak-style */
+const path = require('path');
+/* eslint-disable */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  devtool: 'inline-source-map',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].js',
```

Comment:
[OPTIONAL] Please give the output js file a proper name like `bundle.js` or `main.js` instead of `[name].js`.  The `[name].js` is just an example, it's not the conventional way it should be named.